docs/inference/switch-to-brev-nemotron-120b.md

```
@@ -0,0 +1,220 @@
+---
+
+## Step 4 — Build and verify
+
+```bash
+docker build -t nemoclaw .
+docker run --rm nemoclaw -c 'openclaw models list'
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)